<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8AF4MIK" failures="1" tests="1" name="LoginTest_Chrome" time="5.235" errors="0" timestamp="2024-11-19T09:53:50 IST">
  <testcase classname="testPackage.TC_002_LogOut" name="validateLogOut" time="1.882">
    <failure type="java.lang.AssertionError" message="expected [https://allteacher.navadhiti.com/dashboar] but found [https://allteacher.navadhiti.com/dashboard]">
      <![CDATA[java.lang.AssertionError: expected [https://allteacher.navadhiti.com/dashboar] but found [https://allteacher.navadhiti.com/dashboard]
at pageObjects.PG_002_LogOut.verify_the_Results(PG_002_LogOut.java:39)
at testPackage.TC_002_LogOut.validateLogOut(TC_002_LogOut.java:39)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- validateLogOut -->
</testsuite> <!-- LoginTest_Chrome -->
